src/test/suite/fixtures/recorded/languages/html/clearStartTag.yml

```diff
@@ -0,0 +1,29 @@
+languageId: html
```

Comment:
I wonder if "clear" should leave the angles. Can't decide

Reply:
Take and clear always have the same range so far. I don't want to change that